cogs/modping.py

```diff
@@ -0,0 +1,32 @@
+from typing import Optional
+
+import nextcord
+from nextcord.ext import commands
+
+
+class Confirm(nextcord.ui.View):
+    @nextcord.ui.button(label='Confirm', style=nextcord.ButtonStyle.green, custom_id="modpingconfirm")
+    async def confirm(self, button: nextcord.ui.Button, interaction: nextcord.Interaction):
+        await interaction.send("<@&891630950813925377> has been requested by " + interaction.user.mention)
```

Comment:
Maybe this role id could be in env vars?

Reply:
*And use f-strings*